src/app/profile/yourGiving/editRecurringGifts/editRecurringGifts.modal.component.js

```diff
@@ -2,11 +2,11 @@ import angular from 'angular';
 import some from 'lodash/some';
 
 import loadingComponent from 'common/components/loading/loading.component';
-import step0AddUpdatePaymentMethod from './step0/addUpdatePaymentMethod.component';
+import step0AddUpdatePaymentMethod from './step0/addUpdatePaymentMethod.component.js';
```

Comment:
Intellij does that when refactoring automatically (I dragged the files to a new folder). I'll fix it.